src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/GetCaseResponse.java

```diff
@@ -62,8 +63,8 @@ public class GetCaseResponse {
     @JsonProperty("stages")
     private List<SimpleStageDto> stages;
 
-    @JsonProperty("completed")
-    private Boolean completed;
+    @JsonProperty("dateCompleted")
+    private LocalDateTime dateCompleted;
```

Comment:
It is good that we're adding dateCompleted to the response, but we still need to keep completed in the response JSON as well until clients of the API have been updated to use dateCompleted. This can be done by adding a getCompleted method with `@JsonProperty("completed")`:

```
@JsonProperty("completed")
public Boolean getCompleted() {
    return dateCompleted != null;
}
```